service_info_cms/views.py

```diff
@@ -0,0 +1,17 @@
+from cms.models.pagemodel import Page
+from django.shortcuts import redirect
+from .models import PageRating
+
+
+def update_page_rating(request):
+    if request.method == 'POST':
+        rating = int(request.POST.get("rating", ""))
+        page_id = int(request.POST.get("page_id", ""))
+        return_url = request.POST.get("return_url", "")
+        if all([rating, page_id, return_url]):
```

Comment:
another case to catch is up above, if you try to do int("") or int("not-a-good-number')